### PR TITLE
PLDM: FFDC enhancement in the R/R Path

### PR DESCRIPTION
**Pull request by sagisin**
Signed-off-by: Sagar Srinivas <sagar.srinivas@ibm.com>

### CODE REVIEW

File: host-bmc/host_pdr_handler.cpp [hunk 1]
```diff
@@ -1022,6 +1025,7 @@ void HostPDRHandler::_setHostSensorState()
                 std::cerr << "Failed to "
                              "encode_get_state_sensor_readings_req, rc = "
                           << rc << std::endl;
+                std::cerr << "SensorID=" << sensorId << std::endl;
```
Comment (manojkiraneda) [inline host-bmc/host_pdr_handler.cpp:1028]: add this to the previous line  (1025 line )

File: host-bmc/host_pdr_handler.cpp [hunk 2]
```diff
@@ -853,6 +853,8 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
         this->createDbusObjects(fruRecordSetPDRs);
         if (isHostUp())
         {
+            std::cout
+                << "PDR exchange with Host is done post reset, Host is up, setting the host sensor state\n";
```
Comment (manojkiraneda) [inline host-bmc/host_pdr_handler.cpp:857]: change it to : "Host is UP & Completed the PDR Exchange with host"

File: host-bmc/host_pdr_handler.cpp [hunk 3]
```diff
@@ -975,6 +977,7 @@ void HostPDRHandler::setHostSensorState()
 {
     sensorIndex = stateSensorPDRs.begin();
     _setHostSensorState();
+    std::cout << "All the state sensor reading post Reset/Reload is complete\n";
```
Comment (manojkiraneda) [inline host-bmc/host_pdr_handler.cpp:980]: "Completed get_state_sensor_reading commands on all the host sensors"

Comment (manojkiraneda) [reply host-bmc/host_pdr_handler.cpp:980]: this does not seem to be the correct - place

File: host-bmc/host_pdr_handler.cpp [hunk 4]
```diff
@@ -1139,6 +1146,7 @@ void HostPDRHandler::_setHostSensorState()
                 std::cerr << " Failed to send request to get State sensor "
                              "reading on Host "
                           << std::endl;
+                std::cerr << "SensorID=" << sensorId << std::endl;
```
[the remark this answers is not in the document]
Comment (sagisin) [reply host-bmc/host_pdr_handler.cpp]: done

File: host-bmc/host_pdr_handler.cpp [hunk 5]
```diff
@@ -1109,6 +1115,7 @@ void HostPDRHandler::_setHostSensorState()
                     {
                         std::cerr << " Error invalid_data, Invalid event state"
                                   << std::endl;
+                        std::cerr << "SensorID=" << sensorId << std::endl;
```
[the remark this answers is not in the document]
Comment (sagisin) [reply host-bmc/host_pdr_handler.cpp]: done

File: host-bmc/host_pdr_handler.cpp (hunk 3 above)
Comment (sagisin) [reply host-bmc/host_pdr_handler.cpp]: done

File: host-bmc/host_pdr_handler.cpp [hunk 6]
```diff
@@ -1035,8 +1036,10 @@ void HostPDRHandler::_setHostSensorState()
                                                            size_t respMsgLen) {
                 if (response == nullptr || !respMsgLen)
                 {
+                    std::cerr << "SensorID=" << sensorId << std::endl;
                     std::cerr << "Failed to receive response for "
-                                 "getStateSensorReading command \n";
+                                 "getStateSensorReading command, SensorID="
+                              << sensorId << std::endl;
```
[the remark this answers is not in the document]
Comment (sagisin) [reply host-bmc/host_pdr_handler.cpp:1042]: yes

[the remark this answers is not in the document]
Comment (nmuruli) [reply host-bmc/host_pdr_handler.cpp:1042]: Line 1039 is redundant then correct

[the remark this answers is not in the document]
Comment (sagisin) [reply host-bmc/host_pdr_handler.cpp:1042]: got it

**Review by manojkiraneda — APPROVED** [review]

**Pull request merged**